website/components/features-list/index.jsx

```diff
@@ -0,0 +1,22 @@
+import Feature from './feature'
+import s from './style.module.css'
+
+export default function FeaturesList({ title, features }) {
+  return (
+    <div
+      className={s.featureListContainer}
+      style={{
+        backgroundImage: `url(${require('./images/top-right-design.svg')}), url(${require('./images/bottom-left-design.svg')})`,
+      }}
+    >
+      <div className="g-grid-container">
```

Comment:
I try to avoid using global class names in a CSS module.  It slightly breaks encapsulation, and as a result of that I have a little trick to self-document this.

I'll DM ya a slack thread and context on this where we had a conversation as a team about this, but largely, I'll use a CSS module class name like `s.contentWrapper` here, and then define content wrapper as such:

```css
.contentWrapper {
  composes: g-grid-container from global;
}
```

This "from global" acts as a greppable artifact explicitly acknowledging that CSS module usage is being broken, where as the `g-grid-container` class usage relies on convention, which is not always followed (thus not absolutely greppable).

Reply:
Ah, I saw you do this elsewhere, so you probably get it!